app/src/strategies/tokenStrategy.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+"use strict";
+
+import type { SessionStorageInterface } from "../services/sessionStorageInterface";
+import container from "../container";
+
+const Strategy = require("passport-http-bearer");
+
+const tokenStrategy = new Strategy(function(token, done) {
+  const sessionStorage = (container.resolve(
+    "sessionStorage"
```

Comment:
question about this IoC approach, how can we check that `container.resolve` gets called with valid identifiers? (e.g. catching typos in module name)

Reply:
also, how can we verify that `sessionStorage` has been instantiated before `tokenStrategy` is there a concept of dependency?